### PR TITLE
Dylovene counters sleeping chems

### PR DESCRIPTION
**Pull request by Torque4607**
Now Dylovene will remove soporific and chloral hydrate from the bloodstream, same as it does with zombie powder. Counters sleeping meds, makes them less of a pain once you're in a doctor's hands.

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
As some one who uses sleepy chems, and has no option other than to sit there and wait for them to wake up, this is great.

**Comment by Torque4607** [comment]
Also makes it easier to recover when Urist McGreyshit makes ten soporific syringes and goes on a stabbing spree.

**Comment by chaoko99** [comment]
Interestingly, it was already supposed to do this.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 1]
```diff
@@ -93,6 +93,8 @@
 	var/remove_generic = 1
 	var/list/remove_toxins = list(
 		/datum/reagent/toxin/zombiepowder
```
Comment (Andromeda-K22) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: wait a sec you missed a comma here

**Pull request merged**